dev-tools/reconfigurator-cli/tests/output/cmds-example-stdout

```diff
@@ -694,16 +694,16 @@ external DNS:
 from: blueprint 02697f74-b14a-4418-90f0-c28b2a3a6aa9
 to:   blueprint 86db3308-f817-4626-8838-4085949a6a41
 
- MODIFIED SLEDS:
+ ADDED SLEDS:
```

Comment:
This isn't that worrisome but I'm not clear on why this happened.  This test doesn't appear to explicitly add any sleds.